Bottom: 295caca9f125c4d82621e0492cdf1bef38ff7f29
Top:    365ecb4fe4a9c2f011160c0843bf979537e0202f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:41:46 +0000

Turn off debug hacks in object generation


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 0723343..3468aec 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1483,9 +1483,9 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   destroy_appearance (copper_appearance);
 
   /* DEBUG */
-  poly_M_Copy0 (&PCB->Data->outline, info.poly);
-  PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  poly_M_Copy0 (&PCB->Data->outline, info.poly);
+//  PCB->Data->outline_valid = true;
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
